packages/twenty-server/src/engine/integrations/file-storage/drivers/s3.driver.ts

```diff
@@ -70,6 +72,31 @@ export class S3Driver implements StorageDriver {
     return Readable.from(file.Body);
   }
 
+  async move(params: {
```

Comment:
Ive been able to test the local driver, but cant tell whether AWS will work or not. ChatGPT thinks it would :D